### ISSUE TITLE
3 Level Recon 

### ISSUE THREAD

**Issue by freyxfi** [freyxfi/BugRecon#6]
If you run the script you will see I have developed 3 level in the recon  Shallow, Medium and deep I will explain more this on discord if you connect with me my username is xfi1337

**Comment by bakayu** [comment]
Hey! I would like to connect to discuss about this project, sent you a DM request on discord :)

**Comment by freyxfi** [comment]
Connected :) And thanks for coming forward

**Comment by bakayu** [comment]
Hey! looks like #17 did most of my work
I found an issue where the help menu was not popping up if "-h/--help" argument was passed, I have fixed it in #21.
Also I have added a new interactive mode that opens up when no arguments are passed, and the script can still be run directly when domain and depth-level arguments are passed properly.

![image](https://github.com/user-attachments/assets/51f29f04-c0fc-4dfd-81aa-84f914e86bc3)
![image](https://github.com/user-attachments/assets/04dbc348-7079-42bf-8d77-6059120dcaac)
![image](https://github.com/user-attachments/assets/3a3275ac-4f97-438d-8178-d2584d86233e)
